### PR TITLE
Fix #517 Configurable Docker Compose exposed ports

### LINKED ISSUE #517: [REQUEST] Allow to configure Docker Compose exposed ports
**Is your feature request related to a problem? Please describe.**
Default exposed ports (80, 3000, 27017) of the services may be in conflict with the host services.
E.g. an existing MongoDB instance.

**Describe the solution you'd like**
Allow to configure at least the exposed ports via environment variables.
E.g. 
```
environment:
  DATABASE_PORT: "${DATABASE_PORT:-27017}"
```

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by MaurizioCasciano**
This PR fixes #517 by allowing the use of environment variables in Docker Compose to configure the exposed ports.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,37 +3,44 @@ services:
...
-      - "80:80"
+      - "${FRONTEND_PORT:-80}:80"
     volumes:
       - ./docker-compose-resources/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
 
   rest-api:
     container_name: attack-workbench-rest-api
     build: ../attack-workbench-rest-api
+    environment:
+      BACKEND_PORT: "${BACKEND_PORT:-3000}"
```
Comment (seansica) [inline docker-compose.yml:19]: Please remove this line.

File: docker-compose.yml [hunk 2]
```diff
@@ -3,37 +3,44 @@ services:
...
+      DATABASE_PORT: "${DATABASE_PORT:-27017}"
```
Comment (seansica) [inline docker-compose.yml:20]: Please remove this line.

**Review by seansica — CHANGES_REQUESTED** [review]
Please remove unnecessary environment variables from persisting in the container. The rest looks okay.

**Review by Wheeler69 — COMMENTED** [review]
nice

**Comment by seansica** [comment]
This compose file has been superseded by the template available in the newer [Workbench Deployment](https://github.com/mitre-attack/attack-workbench-deployment) repo.

Thank you for your contribution.

**Pull request closed without merging**